BUILD.md

```diff
@@ -16,6 +16,12 @@ brew install libtool
 brew install gmp
 ```
 
+For Linux, our tests use ktor-client which depends on [ktor-client-curl-linuxx64](https://mvnrepository.com/artifact/io.ktor/ktor-client-curl-linuxx64) so you should install:
+
+```sh
+apt install libcurl4-openssh-dev
```

Comment:
ktor has documented what is needed for linux here:https://ktor.io/docs/client-engines.html#curl and they suggest installing the gnutls version of libcurl4, we should add a link to this.

Reply:
Fixed in [806c569](https://github.com/ACINQ/lightning-kmp/pull/702/commits/806c5690847ef3262f7e5d5696fdccefc0807ae9). How does that look?